Corona-Warn-App/src/main/java/de/rki/coronawarnapp/vaccination/core/qrcode/VaccinationCertificateV1Parser.kt

```diff
@@ -0,0 +1,72 @@
+package de.rki.coronawarnapp.vaccination.core.qrcode
+
+import com.google.gson.Gson
+import com.upokecenter.cbor.CBORObject
+import de.rki.coronawarnapp.util.serialization.fromJson
+import de.rki.coronawarnapp.vaccination.core.qrcode.InvalidHealthCertificateException.ErrorCode.HC_CBOR_DECODING_FAILED
+import de.rki.coronawarnapp.vaccination.core.qrcode.InvalidHealthCertificateException.ErrorCode.VC_NO_VACCINATION_ENTRY
+import org.joda.time.Instant
+import javax.inject.Inject
+
+class VaccinationCertificateV1Parser @Inject constructor() {
+
+    private val keyEuDgcV1 = CBORObject.FromObject(1)
```

Comment:
Could we make this part of a companion object, as we may have multiple instances of `VaccinationCertificateV1Parser`, but these indexes seem static.